src/main/java/seedu/address/commons/events/ui/CardPanelSelectionChangedEvent.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.commons.events.ui;
+
+import seedu.address.commons.events.BaseEvent;
+import seedu.address.ui.CardCard;
+
+/**
+ * Represents a selection change in the Card List Panel
+ */
+public class CardPanelSelectionChangedEvent extends BaseEvent {
```

Comment:
naming of the UI components is quite annoying because the term `card` has 2 meanings, and we end up with terms ilke `CardCard`. Is this component going to only be for `CardCard`s?

Reply:
Yeah it's designed only for `CardCard`s for the back-of-card panel to subscribe to selection events.

Reply:
could you rename this to `CardListPanelSelectionChangedEvent`?